pacman-nodejs-app/tools/migrate/kmt.sh

```diff
@@ -251,18 +265,17 @@ function verify_deployments_ready {
 
         for d in ${deployments}; do
             # Filter deployment whose replica counts do not match i.e. creating
-            local not_ready=$(kubectl get deploy/$d -o json | \
-                jq '.status | select(.availableReplicas != .readyReplicas) and select(.readyReplicas != .replicas)')
-
-            # If we determine that deployment is not ready, try again.
-            if [[ ${not_ready} == true ]]; then
-                all_ready=false
-                break
-            fi
-        done
-
-        (( timeout -= 5 ))
-        sleep 5
+	    local not_ready=$(kubectl get deploy/$d -o json | \
```

Comment:
It doesn't look like you've changed anything here except for the indentation is no longer correct.